app/src/main/java/com/technopark/hw_1/SecondFragment.java

```diff
@@ -0,0 +1,34 @@
+package com.technopark.hw_1;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+public class SecondFragment extends Fragment {
```

Comment:
The same about naming, let's add some context to the name of the fragment, e.g. NumberFragment